src/py/backend/backend.py

```diff
@@ -84,10 +90,19 @@ def prepare(cls, model, device=None, **kwargs):
                 raise RuntimeError(
                     "Incompatible device expected '{0}', got '{1}'".format(
                         device, get_device()))
+            old_stdout = sys.stdout
+            sys.stdout = mystdout = StringIO()
```

Comment:
Cant we just write to the strings directly? Doesn't something like `"Program = {}".format(inf)` work?

Reply:
fixed.